docs/04-enjin-blockchain/02-enjin-relaychain/01-validator-staking.md

```diff
@@ -4,8 +4,10 @@ slug: "validator-staking"
 ---
 This is an overview of staking on Enjin Relaychain, known as liquid staking.
 
-:::info The Enjin Blockchain Console
-Use [console.enjin.io](https://console.enjin.io/) to use the user interface referenced in this document.
+:::info Staking Interfaces
+- **Enjin Console:** You can use [console.enjin.io](https://console.enjin.io/) to access the full user interface referenced in this document.
+  - *Note:* Ensure you connect to the **Enjin Relaychain** network in the top left corner, as staking takes place on the Relaychain.
+- **NFT.io:** Alternatively, you can use **[NFT.io](https://nft.io/staking)** for a simplified experience. While it offers fewer advanced features than the Console, it provides a convenient, user-friendly interface to manage your stake and nomination pools.
```

Comment:
Prioritise the simplified experience (via `nft.io`) over using the more advanced and complicated experience (console). 

I would be in favour of rewriting the guide (where required) to favour the simplified experience as it's natively maintained by us and optimised for Enjin Staking.